backend/src/main/java/com/redhat/cloud/notifications/routers/models/behaviorgroup/UpdateBehaviorGroupRequest.java

```diff
@@ -29,4 +32,16 @@ public class UpdateBehaviorGroupRequest {
      * This will effectively set the linked event types..
      */
     public Set<UUID> eventTypeIds;
+
+    /**
+     * Validates that the display name is not blank when it is provided. Since
+     * this is a potential payload for the "update" operation, clients might
+     * not even send the display name if they don't want to update it. That is
+     * why we let "null" display names go through.
+     * @return true if the display name is not null, but it is blank.
+     */
+    @AssertFalse(message = "the display name cannot be empty")
+    private boolean isDisplayNameNotNullAndBlank() {
+        return this.displayName != null && this.displayName.isBlank();
+    }
```

Comment:
Is there any reason to do this instead of using the `@NotBlank` annotation on the field?

Reply:
By setting the `@NotBlank` annotation directly I would force the clients to always send tie `displayName`. For whatever reason I thought we wanted to let them not specify the `displayName` if they don't want to? :thinking: 

Either way, if you're okay with making them send the `displayName` every time I don't have any problem in simplifying this to a `@NotBlank` annotation.

Reply:
My bad, I just realized that the display name is optional in `UpdateBehaviorGroupRequest` and is only updated in the DB if the field is not `null`. Your check is perfect 👍

Reply:
I should have read the comment you added first 😄

Reply:
Hahaha no worries at all :smile: